package.json

```diff
@@ -51,6 +51,7 @@
     "eslint": "~2.2.0",
     "eslint-config-standard": "^5.1.0",
     "eslint-plugin-babel": "^3.1.0",
+    "eslint-plugin-markdown": "^1.0.0-beta.1",
```

Comment:
i've had enough problems dealing with betas w/ babel-eslint. i'd say to at least wait until the stable version comes out.

but a lot of these changes are kind of annoying as pointed out by @PlasmaPower